### ISSUE TITLE
Update Assignments

### ISSUE THREAD

**Issue by dojo-json** [CodingDojo-MEAN-05-18/SethSondenaa#1]
Please push your latest assignments, then close this issue.

**Comment by sethsondenaa** [comment]
I added them.

Thanks,
Seth

**Comment by dojo-json** [comment]
Excellent, thanks.

**Comment by dojo-json** [comment]
Assignment push needed again.

**Comment by sethsondenaa** [comment]
I am working on getting some new assignments up. My mac was questionable if
it would last throughout the class, and it crashed a couple weeks ago. I
had to wait a bit until I was able to get a new computer, which I got set
up an running last week. Now I have ben trying to catch back up. I have
been working on adding the interests field for a day and a half now from
the first mongodb assignment. Still working on it.

I have been keeping up on lectures during this time and am hoping to crank
out assignments, but have the hang up described above. The good news is
that I now have exposure to Mac and PC computer during the coding class.

Thanks,
Seth
